portafly/src/i18n/locales/en/audience/accounts/listing.yml

```diff
@@ -0,0 +1,108 @@
+---
+#--------------------------Path: Audience > Accounts > Listing ------------------------------------#
+
+# Page details
+
+title_page: Accounts | Listing
+title_page_description: Body title for a page aimed to list all the accounts, as well as perform individual or batch actions upon the accounts.
+
+# Top area
+
+create_account_button: Create account
+create_account_button_description: Button link to create an account
+export_accounts_button: Export all accounts
+export_accounts_button_description: Button link to export all the accounts.
+
+# Main area
+
+## Toolbar
+actions_empty_label: Select accounts first to enable actions
+actions_empty_label_description: Admonition that displays inside the actions dropdown when no items are selected
+actions_dropdownlist: Actions
+actions_dropdownlist_description: String to list all the available actions.
+actions_dropdownlist_options:
+  send_email: Send email
+  change_state: Change state
+
+accounts_filter: Group/organization
```

Comment:
maybe we should be explicit and call this a default value?

Reply:
It is a default value assigned to the accounts_filter.

Reply:
@porueesq sure, I was suggesting to give it a more intuitive name, eg `accounts_filter_default`

Reply:
We can talk about adding `_default` suffixes in the near future. I'll keep as is to avoid more changes.